content/08-cardano-sidechains/03-proof-of-concept-evm-sidechain/04-deploy-smart-contracts/02-setup-development.mdx

```diff
@@ -78,7 +78,7 @@ When the contract is compiled, you can view contract ABI, Bytecode, and more inf
 
 To verify that the smart contract was deployed on the sidechain testnet you can:
 
-- Search the transaction hash or contract address on the EVM sidechain [testnet explorer](https://explorer.sidechain.evmtestnet.iohkdev.io/).
+- Search the transaction hash or contract address on the EVM sidechain [testnet explorer](https://github.com/input-output-hk/blockscout).
```

Comment:
@NeilBurgess42 
Replace ''Search the transaction hash or contract address on the EVM sidechain [testnet explorer]'' into
''Search the transaction hash or contract address on a blockexplorer. To setup: https://github.com/input-output-hk/blockscout''